appium/README.md

```diff
@@ -0,0 +1,16 @@
+# appium project
+
+Project to run Appium tests together with WebdriverIO for:
+
+- iOS/Android Native Apps
+
+## Before running tests, please create a `./apps` directory, download the app and move the app file into that directory
```

Comment:
Could there be a command to automate this? Maybe using fastlane? To create a build and immediately test it (it will be needed for CI anyway)

Reply:
yeah, we could make comand to copy .app file